src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy

```diff
@@ -118,14 +118,15 @@ class UnityPlugin implements Plugin<Project> {
                 ConventionMapping taskConventionMapping = task.conventionMapping
                 applyBaseConvention(taskConventionMapping, extension)
                 taskConventionMapping.logFile = {
-                    project.file("${project.buildDir}/logs/${task.name}.log")
+                    project.file("${project.buildDir}/logs/${task.getLogCategory()}/${task.name}.log")
                 }
             }
         })
     }
 
     static void applyBaseConvention(ConventionMapping taskConventionMapping, UnityPluginExtension extension) {
         taskConventionMapping.map "unityPath", { extension.unityPath }
+        taskConventionMapping.logCategory = { extension.logCategory }
```

Comment:
We should check if we need to wrap values in closures with this mapping style.

Reply:
It is needed:

```java
public void propertyMissing(String name, Object value) {
        if (value instanceof Closure) {
            map(name, (Closure) value);
        } else {
            throw new MissingPropertyException(name, getClass());
        }
    }
```